src/components/common/PeopleAndRoles/OrgPerson.vue

```diff
@@ -572,18 +572,18 @@ export default class OrgPerson extends Mixins(CommonMixin, OrgPersonMixin) {
       // populate mailing address
       // NB: add optional fields for later comparison
       this.inProgressMailingAddress = {
-        ...this.orgPerson.mailingAddress,
         deliveryInstructions: null,
-        streetAddressAdditional: ''
+        streetAddressAdditional: '',
+        ...this.orgPerson.mailingAddress
       }
```

Comment:
This code constructs a new object from the properties inside the curly braces. (`...this.orgPerson.mailingAddress` spreads out its properties, to be included in the new object.)

The bug is that later properties override earlier properties: in the old code, mailingAddress was spread and then deliveryInstructions and streetAddressAdditional were set... thus overriding those fields in the original mailing address.